<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>488</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Admin Panel </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(25, 54, 89, 100);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(105, 118, 132, 200)</string>
    </property>
    <property name="text">
     <string>Welcome!  Shop Lens v0.1 Admin Operations :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
background-color:rgba(2, 65, 118, 255);
color:rgba(255, 255, 255, 200);
border-radius:5px;
}
QPushButton#pushButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2, 65, 118, 100);
background-position:calc(100% - 10px)center;
}
QPushButton#pushButton:hover{
background-color:rgba(2, 65, 118, 200);
}</string>
    </property>
    <property name="text">
     <string>Add Products</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_2{
background-color:rgba(2, 65, 118, 255);
color:rgba(255, 255, 255, 200);
border-radius:5px;
}
QPushButton#pushButton_2:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2, 65, 118, 100);
background-position:calc(100% - 10px)center;
}
QPushButton#pushButton_2:hover{
background-color:rgba(2, 65, 118, 200);
}</string>
    </property>
    <property name="text">
     <string>Sales Analytics</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_3{
background-color:rgba(2, 65, 118, 255);
color:rgba(255, 255, 255, 200);
border-radius:5px;
}
QPushButton#pushButton_3:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2, 65, 118, 100);
background-position:calc(100% - 10px)center;
}
QPushButton#pushButton_3:hover{
background-color:rgba(2, 65, 118, 200);
}</string>
    </property>
    <property name="text">
     <string>Payment Logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_4{
background-color:rgba(2, 65, 118, 255);
color:rgba(255, 255, 255, 200);
border-radius:5px;
}
QPushButton#pushButton_4:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2, 65, 118, 100);
background-position:calc(100% - 10px)center;
}
QPushButton#pushButton_4:hover{
background-color:rgba(2, 65, 118, 200);
}</string>
    </property>
    <property name="text">
     <string>Check Stock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_5{
background-color:rgba(2, 65, 118, 255);
color:rgba(255, 255, 255, 200);
border-radius:5px;
}
QPushButton#pushButton_5:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2, 65, 118, 100);
background-position:calc(100% - 10px)center;
}
QPushButton#pushButton_5:hover{
background-color:rgba(2, 65, 118, 200);
}</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
